pkg/services/resource.go

```diff
@@ -48,20 +48,26 @@ type sqlResourceService struct {
 	lockFactory db.LockFactory
 	resourceDao dao.ResourceDao
 	events      EventService
+	generic     GenericService
 }
 
 func (s *sqlResourceService) Get(ctx context.Context, id string) (*api.Resource, *errors.ServiceError) {
 	resource, err := s.resourceDao.Get(ctx, id)
 	if err != nil {
 		return nil, handleGetError("Resource", "id", id, err)
 	}
-	return resource, nil
-}
 
-func (s *sqlResourceService) GetBundle(ctx context.Context, id string) (*api.Resource, *errors.ServiceError) {
-	resource, err := s.resourceDao.GetBundle(ctx, id)
-	if err != nil {
-		return nil, handleGetError("Resource Bundle", "id", id, err)
+	// fill back the creationTimestamp and deletionTimestamp from resource meta to work metadata if it exists
```

Comment:
using a function to wrap this, then we can reuse it in the ListByArgs?

Reply:
done.